packages/terra-consumer-nav/src/components/quick-links/QuickLinks.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import './QuickLinks.scss';
+
+const propTypes = {
+  quickLinks: PropTypes.arrayOf(
+    PropTypes.shape({
+      slug: PropTypes.string,
+      nav_type: PropTypes.string,
+      uri: PropTypes.string.isRequired,
+      text: PropTypes.string.isRequired,
+      /*
+      TODO: Need to figure out what we are doing with icons
+
+      icon: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.element,
+      ]),
+      */
+    }),
+  ),
+  maxListSize: PropTypes.number,
+};
+
+const defaultProps = {
+  quickLinks: [],
+  maxListSize: 4,
```

Comment:
We dont' need to manage this here... this should be managed in wherever creates/persists the config on the server